docker/Dockerfile.ci

```diff
@@ -77,6 +85,22 @@ COPY environment.yml /root/code/garage/environment.yml
 RUN conda env create -f /root/code/garage/environment.yml && \
   rm -rf /opt/conda/pkgs/*
 
+# Roboschool in conda
+RUN git clone --depth 1 https://github.com/openai/roboschool.git && \
```

Comment:
Please always clone into explicit absolute paths for docker. Usually into /root/